app/services/cancel_rdv_by_agent_service.rb

```
@@ -0,0 +1,15 @@
+class CancelRdvByAgentService
+  attr_reader :rdv
+
+  def initialize(rdv)
+    @rdv = rdv
+  end
+
+  def perform
+    @rdv.update(status: :excused, cancelled_at: Time.now)
```

use `update!` or else it could fail silently